frontend/src/components/common/form/styles.ts

```diff
@@ -68,7 +68,6 @@ export const InlineCol = styled(Col)`
 export const InlineFastDatePicker = styled(FastDatePicker)`
   display: flex;
   align-items: baseline;
-  gap: 0.8rem;
```

Comment:
This will probably affect a number of form, have you regression validated this?

Reply:
I took a look at them, the problem was that this is the extra space that was showing up on the forms when they where stacked